.github/automation/commit-msg-check.py

```diff
@@ -67,7 +58,8 @@ def main():
     head: str = args.head
 
     commit_range = base + ".." + head
-    messages = subprocess.run(["git", "rev-list", "--format=oneline", commit_range], capture_output=True, text=True).stdout
+    messages = subprocess.run(["git", "rev-list", "--format=oneline",
+               commit_range], capture_output=True, text=True).stdout
```

Comment:
(nit) according to [peps8](https://peps.python.org/pep-0008/#indentation), arguments should 
- either start on the next line with 4 spaces offset, 
- or start on same line but upon line break be aligned with the opening bracket

Reply:
we need clang-format but for Python...